### ISSUE TITLE
Arbitrary mean functions

### ISSUE THREAD

**Issue by stefanocovino** [STOR-i/GaussianProcesses.jl#231]
Dear friends,

Is there a way to use arbitrary mean functions with this package?  I bet it is definitely possible to clone the code for the already available mean functions. I actually wonder whether there could be a more generic method.

Thanks,
  Stefano

**Comment by maximerischard** [comment]
Yes, absolutely, you can write your own mean functions. You can look at the standard ones provided in the package and use them as a template. No need to modify the package to do this.
